### ISSUE TITLE
decommission node console

### ISSUE THREAD

**Issue by qrkourier** [openziti/ziti-console#447]
1. delete CI jobs that test and build the node.js web application server
2. delete mentions in scripts
3. delete mentions in docs

**Comment by qrkourier** [comment]
@rgallettonf @dovholuknf Do you feel ready for this housekeeping task to happen?

**Comment by dovholuknf** [comment]
no. there's doc that needs to happen first. communication that needs to happen, waiting period etc... there's no need to rush this. it can stick around for a while

**Comment by qrkourier** [comment]
4. delete netfoundry/ziti-console docker stuff from package.json
